src/services/chat/prepopulated.js

```diff
@@ -41,17 +41,17 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser
+                  ...ownUser.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
               },
               {
                 user: {
-                  ...user
+                  ...user.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: user?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updated_at` field is being set to the current date and time. This is fine if you want to record the exact time when this operation is performed. However, if you want to keep the original `updated_at` value from the user object, you would need to adjust this.


<!-- This is an auto-generated comment by OSS CodeRabbit -->